src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/styles/AppStyles.kt

```diff
@@ -64,6 +70,39 @@ class AppStyles : Stylesheet() {
                 fontWeight = FontWeight.BLACK
             }
         }
-    }
 
+        wizardCard {
+            prefWidth = 300.px
+            prefHeight = 320.px
+            backgroundColor += c(UIColorsObject.Colors["base"])
+            padding = box(10.px)
+            backgroundRadius += box(10.px)
+            spacing = 10.px
+            wizardCardGraphicsContainer {
+                backgroundRadius += box(10.px)
+                backgroundColor += c(UIColorsObject.Colors["base"])
+            }
+            label {
+                textFill = Color.BLACK
+            }
+            s(".jfx-button") {
```

Comment:
Move this to the stylesheet?

Reply:
I think rather have this here and leave the wizard widget itself less styled. the styling of the widget should be done in the application just so we don't have the widgets be dependent on the app like i said below too